styles/foobar-ui.min.css

```diff
@@ -1 +1 @@
l);padding:var(--space-xs);margin-left:auto}.card-btn{background-color:var(--color-white);color:var(--color-dark);height:var(--space-lg);border:1px solid var(--color-light)}.card-btn:hover{box-shadow:none}.card-btn:not(:last-child){margin-right:var(--space-sm)}.card_with-badge{position:relative}.card_text-only,.card_text-overlay{height:min-content}.card_text-overlay .card-text{position:absolute;bottom:0;backdrop-filter:blur(6px) brightness(80%);color:var(--color-white)}.card_text-overlay .card-title{color:unset}.card_text-overlay .card-description{color:unset}.card_text-overlay .card-action{display:none}.card_text-only .card-media{display:none}.card_text-only .card-action{display:none}.card_horizontal{width:calc(var(--space-sm) * 43.75);position:relative}.card_horizontal .card-content{flex-direction:row}.card_horizontal .card-media{width:50%}.card_horizontal .card-action{position:absolute;bottom:0;right:0}.card_with-shadow{box-shadow:0 2px 1px -1px rgb(0 0 0 / 20%),0 1px 1px 0 rgb(0 0 0 / 14%),0 1px 3px 0 rgb(0 0 0 / 12%)}.img-responsive{max-width:100%;height:auto}.img-round{width:calc(var(--space-xl) * 10);height:calc(var(--space-xl) * 10);object-fit:cover;border-radius:var(--border-radius-circle)}.textbox{margin:var(--space-xs) 0}.textbox label{color:var(--color-dark);display:block;height:var(--space-md);line-height:var(--space-md);padding:0;margin:var(--space-xs);border:0;min-width:0;width:100%}.textbox input{background-color:var(--color-white);color:var(--color-dark);height:var(--space-lg);padding:0 var(--space-sm);margin:var(--space-xs);border:1px solid var(--color-dark);min-width:0;width:100%;border-radius:4px}.textbox-error input{border:1px solid var(--color-primary)}.textbox-validated input{border:1px solid var(--color-success)}.rating{font-size:var(--font-size-md)}.rating .rating-star{cursor:pointer}.rating .icon-filled{color:var(--color-secondary)}.heading-1{font-size:var(--font-size-xl);line-height:var(--line-height-md)}.heading-2{font-size:var(--font-size-lg);line-height:var(--line-height-sm)}.heading-3{font-size:var(--font-size-md);line-height:var(--line-height-md)}.heading-4{font-size:var(--font-size-sm);line-height:var(--line-height-lg)}.heading-5{font-size:var(--font-size-xs);line-height:var(--line-height-xl)}.text-1{font-size:var(--font-size-xl);line-height:var(--line-height-md)}.text-2{font-size:var(--font-size-lg);line-height:var(--line-height-sm)}.text-3{font-size:var(--font-size-md);line-height:var(--line-height-md)}.text-4{font-size:var(--font-size-sm);line-height:var(--line-height-lg)}.text-5{font-size:var(--font-size-xs);line-height:var(--line-height-xl)}.text-small{font-size:calc(var(--font-size-sm) * 5 / 6)}.text-gray{color:var(--color-gray)}.text-center{text-align:center}.text-black{font-weight:var(--font-weight-black)}.text-bold{font-weight:var(--font-weight-bold)}.text-semibold{font-weight:var(--font-weight-semibold)}.text-regular{font-weight:var(--font-weight-regular)}.text-light{font-weight:var(--font-weight-light)}
```

Comment:
please look into the indentation of the file.

Reply:
Hi Vaibhav, it's a generated CSS minified file. No need for indentation here. Thanks for the suggestion!